server/database/models/user.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Why do we have user model here

Reply:
I created it to test the login route with a user seeded into the database

Reply:
So why did you commit it?